SonarQube.TeamBuild.Integration/TfsLegacyCoverageReportProcessor.cs

```diff
@@ -55,7 +55,7 @@ protected override bool TryGetBinaryReportFile(AnalysisConfig config, TeamBuildS
             switch (urls.Count())
             {
                 case 0:
-                    logger.LogMessage(Resources.PROC_DIAG_NoCodeCoverageReportsFound);
+                    logger.LogDebug(Resources.PROC_DIAG_NoCodeCoverageReportsFound);
```

Comment:
Should be the same verbosity as the log message that says we are trying to download coverage reports so the user can see that an action started and how it completed.